DefaultSettings/getDefaultSettings.m

```diff
@@ -132,11 +201,21 @@
         num2str(S.misc.poly_order.lower) '_' num2str(S.misc.poly_order.upper)];
 end
 
+% default coordinate bounds used to approximate musculoskeletal geometry
+if ~isfield(S.misc,'default_msk_geom_bounds')
+    S.misc.default_msk_geom_bounds = 'default_msk_geom_bounds.csv';
```

Comment:
default_msk_geom_bounds.csv not present in this branch?

Reply:
looks like something went wrong with updating this branch from master, should be fixed now.